<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initDialog</class>
 <widget class="QDialog" name="initDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>做一些初始化</string>
  </property>
  <widget class="QLineEdit" name="txtUserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUserName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>26</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名(8个字符)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayerPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>156</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>播放器路径(可以暂时不选择)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIDMPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>156</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>下载器路径(可以暂时不选择)</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnChoosePlayer">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>69</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnChooseIDM">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlayer">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>69</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIDM">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFinished">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
